pywr-schema/src/nodes/core.rs

```diff
@@ -313,15 +313,37 @@ impl TryFrom<OutputNodeV1> for OutputNode {
     }
 }
 
-#[derive(serde::Deserialize, serde::Serialize, Clone, Default)]
+#[derive(serde::Deserialize, serde::Serialize, Clone, PartialEq, Copy, Debug)]
+pub enum StorageInitialVolume {
+    #[serde(alias = "absolute")]
```

Comment:
I don't think we've generally renamed or aliased anything in the new schema. E.g. node types are now required to be capitalised. I wonder if, for consistency, we remove this and use the serde default names. Thoughts?

Reply:
I agree, its better to be consistent